docs/navbar.md

```diff
@@ -0,0 +1,70 @@
+## Navbar component
```

Comment:
I propose the following doc structure:

1. Description of what it does "The navbar component is..."
2. Screenshot.
3. Example PHP code.
4. HTML output.
5. Methods list.

Reply:
done